core/src/main/java/greencity/configuration/SecurityConfig.java

```diff
@@ -130,6 +130,10 @@ protected void configure(HttpSecurity http) throws Exception {
                 UBS_LINK + "/client/**",
                 "/notifications/**")
             .hasAnyRole("USER", "ADMIN")
+            .antMatchers(HttpMethod.PUT,
+                "/notifications/updateTemplateForOTHER",
```

Comment:
can you please use REST standard for URLs? updateTemplateForUther -  not the best name. since it's PUT - it is obvious that it's update